src/js/structs/Service.js

```diff
@@ -146,6 +146,12 @@ module.exports = class Service extends Item {
     let deployments = this.getDeployments();
     let queue = this.getQueue();
 
+    if (this.getInstancesCount() === 0 &&
```

Comment:
Isn't this "duplicating" https://github.com/dcos/dcos-ui/pull/675/files#diff-6e602e3f57a52157c9dba6acf1858558R171?